docs/configuration_reference.md

```diff
@@ -15,9 +15,9 @@ vich_uploader:
     metadata:
         auto_detection: true
         cache: file
-        type: annotation    # or attribute
+        type: attribute    # or annotation, on PHP 7 
```

Comment:
It can still be annotation if annotations are used on PHP8, so it's not really related to the PHP version.

Reply:
The idea is to encourage the use of attributes, so having a default of "attribute" in recent PHP versions. You can still use "annotation" configuration with PHP8, but you need to go explicit